sklearn/utils/testing.py

```diff
@@ -610,6 +568,20 @@ def is_abstract(c):
             return False
         return True
 
+    if include_other is not None:
+        warnings.warn("include_other was deprecated in version 0.19 and will"
```

Comment:
if we're going to deprecate this, rather than remove it, you might as well mention that it no longer does anything (as opposed to deprecation with backwards compatibility)!